draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -274,70 +291,176 @@ The array is created as follows:
+The Issuer MUST hide the original order of the claims in the array. To ensure
+this, it is RECOMMENDED to shuffle the array of hashes (after potentially adding
+decoy digests as described in (#decoy_digests)), e.g., by sorting it
+alphanumerically or randomly. The precise method does not matter as long as it
+does not depend on the original order of elements.
 
-It is the Issuer who decides which claims are selectively disclosable and which are not. However, claims controlling the validity of the SD-JWT, such as `iss`, `exp`, or `nbf` are usually included in plaintext. End-User claims MAY be included as plaintext as well, e.g., if hiding the particular claims from the Verifier does not make sense in the intended use case.
+For example, using the digest of the object property Disclosure created above,
+the Issuer could create the following SD-JWT payload to make `family_name`
+selectively disclosable:
 
-Claims that are not selectively disclosable are included in the SD-JWT in plaintext just as they would be in any other JWT.
+```json
+{
+  "given_name": "Alice",
+  "_sd": ["uutlBuYeMDyjLLTpf6Jxi7yNkEF35jdyWMn9U7b_RYY"]
+}
+```
 
-Selectively disclosable claims are omitted from the SD-JWT. Instead, the digests of the respective Disclosures and potentially decoy digests are contained as an array in a new JWT claim, `_sd`.
+### Array Elements
 
-The `_sd` key MUST refer to an array of strings, each string being a digest of a Disclosure or a decoy digest as described above.
+Digests of Disclosures for array elements are added to the array in the same
+position as the original claim value in the array. For each digest, an object
+of the form `{"...": "<digest>"}` is added to the array. The key MUST be the
+string `...` (three dots). The value MUST be the digest of the Disclosure as
+described in (#hashing_disclosures). There MUST NOT be any other keys in the
+object.
 
-The array MAY be empty in case the Issuer decided not to selectively disclose any of the claims at that level. However, it is RECOMMENDED to omit the `_sd` key in this case to save space.
+For example, using the digest of the array element Disclosure created above,
+the Issuer could create the following SD-JWT payload to make the second element
+of the `nationalities` array selectively disclosable:
 
-The Issuer MUST hide the original order of the claims in the array. To ensure this, it is RECOMMENDED to shuffle the array of hashes, e.g., by sorting it alphanumerically or randomly. The precise method does not matter as long as it does not depend on the original order of elements.
+```json
+{
+  "nationalities":
+    ["DE", {"...": "w0I8EKcdCtUPkGCNUrfwVp2xEgNjtoIDlOxc9-PlOhs"}]
+}
+```
 
-Issuers MUST NOT issue SD-JWTs where
+As described in (#verifier_verification), Verifiers ignore all selectively
```

Comment:
```suggestion
As described in (#verifier_verification), Verifiers MUST ignore all selectively
```

Reply:
This here is intentionally not a normative requirement. We don't need to state the same "MUST" twice.

Reply:
that's fine.